pkg/operator/ceph/file/mds/mds.go

```diff
@@ -120,6 +120,9 @@ func (c *Cluster) Start() error {
 
 	// Always create double the number of metadata servers to have standby mdses available
 	replicas := c.fs.Spec.MetadataServer.ActiveCount * 2
+	if c.fs.Spec.MetadataServer.ActiveStandByCount > 0 {
+		replicas = c.fs.Spec.MetadataServer.ActiveStandByCount
```

Comment:
I believe you want this:
```
replicas = c.fs.Spec.MetadataServer.ActiveCount + c.fs.Spec.MetadataServer.ActiveStandByCount
```

Reply:
Yes. Added the changes. Thanks

Reply:
@travisn Working on the above suggestions.